src/components/tutor-input.cjsx

```diff
@@ -181,10 +178,10 @@ TutorDateInput = React.createClass
         />
     else if @props.disabled and value
       displayValue = value.format(TutorDateFormat)
-      wrapperClasses.push('disabled-datepicker')
+      wrapperClasses += ' disabled-datepicker'
```

Comment:
Yeah, I was on the fence about this.  I didn't add it to the main setting above since I didn't want to duplicate the logic condition in two places.  Your comment forced me to think about it a bit more and realize that we have these nifty things called variables that can hold the results of a condition and be checked in multiple places :)   I've reworked to do that.